packages/gatsby/cache-dir/static-entry.js

```diff
@@ -55,6 +55,11 @@ const createElement = React.createElement
 export default (pagePath, callback) => {
   const pathPrefix = `${__PATH_PREFIX__}/`
 
+  // If enabled, group files in their respective folders otherwise generated
+  // non-static files will be left in the root of the output directory.
+  const pathToGroupStyles = __GROUP_FILES__ ? `styles/` : ``
```

Comment:
how about we use `js`/`css` instead? Those feel more standard to me + they're shorter (save all the bytes).

Reply:
@KyleAMathews agreed. The original names were based off variables in the code. +1 for saving all the bytes.